docs/platform/howto/access-service-logs.rst

```diff
@@ -0,0 +1,24 @@
+Access service logs
+====================
+
+Accessing logs from Aiven service: PostgreSQL®, Kafka®, Cassandra®, Redis®*, InfluxDB®
+
+Occasionally there is a need to inspect logs from Aiven services. For example, to debug query performance or inspecting errors caused by a specific workload.
+
+There are three built-in ways to inspect service logs at Aiven:
+
+* In `Aiven Console <https://console.aiven.io/>`_, when selecting a specific service, under *Logs* tab recent events are available. 
+
+.. Tip::
+
+    Logs can be browsed back in time, but scrolling up several thousand lines is not very convenient.
+
+* :doc:`Aiven CLI </docs/tools/cli>` supports programmatically downloading logs with the :ref:`dedicated command <avn-service-logs>`.
+
+* `Aiven API <https://api.aiven.io/doc/#operation/ProjectGetServiceLogs>`_ endpoint is available for fetching the same information two above methods output, in case programmatic access is needed.
+
+.. Warning::
+
+    Service logs included on the normal service price are stored only for a few days. Unless you are using :doc:`logs integration </docs/integrations>` to another service, older logs are not accessible.
```

Comment:
what is a "normal service price"? can we be more precise?

Reply:
I'm removing the normal service price as it applies to all services actually.